macro/makeGenieEvents.py

```diff
@@ -11,7 +11,8 @@
 # Before runnig this script please run this command in FairShip bash:
 # export GXMLPATH='/eos/experiment/ship/user/aiuliano/GENIE_FNAL_nu_splines'
 # this will disable Genie decays for charm particles and tau
-
+if os.environ.has_key('GXMLPATH'): print('exporting GXMLPATH executed')
+else: print('WARNING: exporting GXMLPATH not executed')
```

Comment:
This would be a good opportunity to use the new FairLogger interface (you'll need to import `logger.warn`:
`warn("GXMLPATH is not set: Genie will decay charm and tau particles, which is usually not the desired behaviour")

Reply:
Do you mean just adding the line you wrote in the if statement  ? Anyway I'll rewrite those line in the way you suggested.

Reply:
Yes, but replacing the print statement.

For the debug one you can use `logger.debug` in a similar way.

`[WARN]` or `[DEBUG]` will be automatically printed in front of the message and the output can be centrally controlled similar to the log messages we have in C++ (e.g DEBUG messages are only printed when the verbosity is changed).

Reply:
Oliver, I changed the file, please warn me if I made a mistake using logging functions.